version

```diff
@@ -0,0 +1 @@
+v1.5.4
```

Comment:
This is required if you installing following manual installation guide.
https://kubernetes.io/docs/getting-started-guides/ubuntu/manual/

Agree with you build adds this file but when you do manual installation this file should exist.